roles/splunk_common/tasks/get_facts.yml

```diff
@@ -24,6 +24,8 @@
   become: yes
   become_user: "{{ privileged_user }}"
   register: pre_existing_splunk_secret
+  become: yes
```

Comment:
Getting some linting errors here due to duplicate keys in this stanza

Reply:
You should also be able to run the checks that CircleCI is doing with linting by running `make lint` in the root of the repo